app/assets/javascripts/admin/addon/controllers/admin-api-keys-new.js

```diff
@@ -20,6 +21,15 @@ export default Controller.extend({
       { id: "all", name: I18n.t("admin.api.all_users") },
       { id: "single", name: I18n.t("admin.api.single_user") },
     ]);
+
+    this.set("scopeModes", [
+      { id: "granular", name: I18n.t("admin.api.scopes.granular") },
+      { id: "read_only", name: I18n.t("admin.api.scopes.read_only") },
+      { id: "global", name: I18n.t("admin.api.scopes.global") },
+    ]);
+
+    this.set("scopeMode", "granular");
```

Comment:
Maybe we leave the default scopeMode empty? It would mean that the long list of scopes won't be shown, and the Scope dropdown will match User Level, i.e. be empty.

Reply:
I implemented these suggestions, thanks for the feedback!